notes/text.md

```diff
@@ -111,3 +111,61 @@ http://www.color-hex.com/color-palette/194
+  </p>
+  <p>
+    Finally, I recommend some kind of red tinted flashlight (my Dad used to make them by rubber banding red cellophane over a flashlight, but you can buy special red LED flashlights now). Even in light polluted areas like mine, it can be really easy to drop lenses and filters in the dark. Unlike white "normal" light, red light won't blind you (and require another 20 minutes to adjust to seeing the night sky properly).
+  </p>
+  <p>
+    Given that I use my IPhone so much as a flashlight (as well as for looking stuff up at the last minute), I've been trying to find some kind of app that would make my IPhone's screens red tinted. However, not surprisingly, that's apparently really hard to do (the app would essentially have to turn all of the other apps' screens red). (Most astronomy apps do a have a red screen setting, at least) I may just have follow my Dad's lead and tape some red cellophane over my phone!
+</p>
+<p class="readless hide"><a href="#">Read Less</a></p>
+-->       </article>
+
+<h2>Light Pollution</h2>
+<p>
+    Unless you literally live in the middle of no where, you are highly likely to experience this.   Unfortunately, there isn't much you can aside from (short term) go somewhere else to look at the stars (long term) advocate for less light pollution (probably a failing effort, sadly). There are telescope filters that can reduce the glare at least somewhat, though.
+</p>
+<!--
+<p>
+    When I lived in Boulder, Colorado, about 50 miles from Denver, you could literally see a reddish glow in the sky at night from all of the light pollution. Given that everything else in Boulder seemed so "natural" (bike trails through parks filled with trees, the Flatirions, Alfalfas...), I always found that incredibly ironic.
+</p>
+<p>
+    What causes light pollution?
+</p>
+<p>
+    The most immediate and obvious answer is light. Street lights, house lights, car lights. In general, a higher population tends to emit more light at night (perhaps most easily seen from a plane at night). However, local environmental conditions can also exacerbate that light - for instance, when the humidity get high enough, individual bits of air vapor (aerosols) literally scatter light across the sky.
+</p>
+<p>
+    There are light pollution charts out there you can use to get an idea of where to go to escape it (they also show how it's gotten worse over time). Ultimately, for those of us who experience it, light pollution can turn even casual observational astronomy from something you do on a "regular" evening into something that requires a special trip and degree of effort, and "backyard astronomy," becomes "back country astronomy."
+</p>
+-->
+</section>
```

Comment:
# FEWD Week #8 Project: Forms

---


### Description


This week, you'll be practicing your form skills: you have the option of adding a form to your final project, finishing the Relaxr form that will be started in class, or building a form for CitiPix. The form you build must include at least three different types of HTML form elements and some light CSS styling. We have given you an example of a form that renders well on the web and mobile devices, as well as an example, with starter code, of a registration form you could build for Relaxr.

Student: ________________________________________

Project URL: ____________________________________

|                                                                                                                                                                |                                |                        |                          |
|----------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------|------------------------|--------------------------|
| Technical Requirements                                                                                                                                         | Does Not Meet Expectations (0) | Meets Expectations (1) | Exceeds Expectations (2) |
| Use at least three different types of HTML5 form elements: ```input```, ```textarea```, ```select```, ```radio``` ```button```, ```dropdown```, ```checkbox``` |                                |       1                 |                          |
| Style forms lightly using CSS                                                                                                                                  |                                |      1                  |                          |
| TOTAL: __2____/4                                                                                                                                                |                                |                        |                          |